templates/dns.yml.erb

```diff
@@ -5,7 +5,7 @@
 #   <%= "dns_" %>dnscmd (Microsoft Windows native implementation)
```

Comment:
Yeah, that's my mistake from 8346937a, there's no need for ERB here or in dhcp.yml.erb!